app/styles/libs/_helpers.scss

```diff
@@ -135,6 +135,24 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   color: #fff;
 }
 
+.social-link-button {
+  width: 50px;
+  height: 50px;
+  max-width: 50px;
+  max-height: 50px;
+}
+
 .w-full {
   width: 100% !important;
 }
+
+.social-link-button {
```

Comment:
Duplicate style. Next time, review the PR yourself once before asking for review